hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/utils/EnrichedSpanCookieUtils.java

```diff
@@ -0,0 +1,115 @@
+package org.hypertrace.traceenricher.enrichedspan.constants.utils;
+
+import static com.google.common.net.HttpHeaders.COOKIE;
+import static org.hypertrace.core.span.constants.v1.Http.HTTP_REQUEST_HEADER;
+
+import com.google.common.base.Splitter;
+import java.net.HttpCookie;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.shared.HexUtils;
+import org.hypertrace.core.span.constants.RawSpanConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EnrichedSpanCookieUtils {
```

Comment:
Can we instead add the util functions inside `HttpSemanticConventionUtils` and an umbrella function under `EnrichedSpanUtils`?

Reply:
Initially, I'd asked Shivansh to introduce 2 new utils @ https://github.com/hypertrace/hypertrace-ingester/pull/338#discussion_r962764685 for maintainable size of classes.

Reply:
I guessed the motivation. We haven't yet decided the entire utils structure in trace enricher. There already happens to be many utils, and we were just sticking to the existing utils flow